moviepy/config.py

```diff
@@ -56,7 +56,11 @@ def try_cmd(cmd):
             IMAGEMAGICK_BINARY = wr.QueryValueEx(key, 'BinPath')[0] + r"\convert.exe"
             key.Close()
         except:
-            IMAGEMAGICK_BINARY = 'unset'
+            try:
+                imagemagick_path   = sp.check_output('dir /B /O-N "C:\Program Files\ImageMagick-*"'                           , shell=True).decode("utf-8").split("\r\n")[0]
```

Comment:
2 concerns. why that long blank after the first argument? 
and what happen on those windows that are not on "C:" and not in english. ? isn't that the "BinPath" requested above, in the line 56, via `wr.QueryValueEx(key, 'BinPath')[0]` ?

Reply:
That long blank is so the line below lines up perfectly to make it more readable.
If you are one of the few who decided to install windows not on C then it wont work.
I have window in german and the folder is called "Programme" but it still works with the english folder name "Program Files"  in the path.
Line 56 yes maybe but it doesn't work (even says so in the installation guide).